webserver/views/datasets.py

```diff
@@ -92,6 +93,41 @@ def view(id):
     )
 
 
+def get_json(id):
+    dataset = get_dataset(id)
+    dataset_clean = {
+        "name": dataset["name"],
+        "description": dataset["description"],
+        "classes": [],
+        "public": dataset["public"],
+    }
+    for cls in dataset["classes"]:
+        dataset_clean["classes"].append({
+            "name": cls["name"],
+            "description": cls["description"],
+            "recordings": cls["recordings"],
+        })
+
+    return dataset_clean
+
+CSV_DIR = "./code/data/files/csv_files"
+
+@datasets_bp.route("/<uuid:dataset_id>/download")
+def download_csv(dataset_id):
+    # return render_template("datasets/accuracy.html")
```

Comment:
be careful when you're copying code from other parts of the project. Make sure you revise your code before committing it